submit_and_compare/submit_and_compare.py

```diff
@@ -1,34 +1,32 @@
-'''  Submit and Compare XBlock main Python class'''
+"""  Submit and Compare XBlock main Python class """
 
+import StringIO
+import textwrap
+import logging
 import pkg_resources
+
 from django.template import Context, Template
 from django.utils.translation import ungettext
-
 from xblock.core import XBlock
 from xblock.fields import Scope, String, List, Float, Integer
 from xblock.fragment import Fragment
-
 from lxml import etree
-from xml.etree import ElementTree as ET
 
-from StringIO import StringIO
```

Comment:
pylint fails if 'from StringIO import StringIO' is below 'import pkg_resources' and if 
'from django.template import Context, Template' is above 'import textwrap' and 'import logging'.  

I played around with the ordering a bit.  There were other exceptions too.